other/plague/plague carrier

```diff
@@ -1,21 +1,26 @@
 **Plague Carrier** | Solo Miscellaneous - Plague Team
 __Basics__ 
-The Plague Carrier wins with the plague bearers and joins a channel with them to communicate securely.
+The Plague Carrier learns which roles will die to the plague.
 __Details__
-The Plague Carrier does not have any special abilities. If somebody tries to copy a plague bearer, a Plague Carrier is created instead. 
+At the start of each night, the Plague Carrier will learn the roles of the players destined to die to the plague at the end of that night. The Plague Carrier does not learn which player has which role. 
```

Comment:
"destined" doesnt really fit the vibe of our descs imo

Reply:
`At the start of each night, the Plague Carrier will learn the roles of the players who will die from the plague at the end of that night.`?

Reply:
it's not a "flavour" word here, it's a meaning word. there is not really another word that conveys the correct meaning except "scheduled" which is, imo, infinitely worse. i understand your dislike but it's ultimately a very small issue that i don't think can be resolved satisfactorally from my end